src/GraphQL/Validation/ValidationError.cs

```diff
@@ -42,7 +42,7 @@ public ValidationError(string originalQuery, string number, string message, para
             nodes?.Apply(n =>
             {
                 _nodes.Add(n);
-                if (n.SourceLocation != null)
+                if (n.SourceLocation != SourceLocation.Empty)
```

Comment:
Are you sure this is correct? An uninitialized `SourceLocation` property will have its values set to 0, not to `SourceLocation.Empty`.

Reply:
🤦